app/helpers/reactjs_helper.rb

```diff
@@ -45,10 +50,20 @@ def js_tags_for(requested_plugins)
     end
   end
 
-  def js_tags_for_global_files(requested_plugins)
+  def global_js_tags(requested_plugins)
     requested_plugins.map do |plugin|
       plugin[:files].map do |file|
-        javascript_include_tag(*webpack_asset_paths(plugin[:id].to_s + ":#{file}", :extension => 'js'), :defer => "defer")
+        javascript_include_tag(*webpack_asset_paths("#{plugin[:id]}:#{file}", :extension => 'js'), :defer => "defer")
+      end
+    end
+  end
+
+  def global_css_tags(requested_plugins)
+    requested_plugins.map do |plugin|
+      plugin[:files].map do |file|
+        stylesheet_link_tag(*webpack_asset_paths("#{plugin[:id]}:#{file}", :extension => 'css'))
+      rescue Webpack::Rails::Manifest::EntryPointMissingError
```

Comment:
note: this is dropping support for ruby 2.4

Reply:
That's not really a problem since we only run on 2.5+

Reply:
I was just trying to be explicit :)